en/docs/tutorials/scenarios/scenario1-create-rest-api.md

```diff
@@ -67,7 +67,10 @@ Before Publishing the API, Coltrain developers want to test this API. WSO2 API M
 
 ## Step 4: Deploy the API in the Gateway
 
-Before starting to test the API, you need to deploy the API in the Gateway. For that, go to **Deploy → Deployment** section and select **Deploy** . This will deploy the API in the Gateway but the API will not be visible to the outside world.
+Before starting to test the API, you need to deploy the API in the Gateway. 
+First you need to select a business plan for the API. For that, go to **Develop → Subscriptions** section and select **Unlimited** and save.
+After that, go to **Deploy → Deployment** section and select **Deploy** . This will deploy the API in the Gateway but the API will not be visible to the outside world.
```

Comment:
```suggestion
After that, go to the **Deploy → Deployment** section and select **Deploy**. This will deploy the API in the Gateway but the API will not be visible to the outside world.
```